src/lib/Layout/PanelSideBarLayout/PanelSideBar/Definitions/PanelSideBarMenuItem.ts

```diff
@@ -0,0 +1,43 @@
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import {ReactNode} from "react";
+
+export type PanelItem<TPanelItem = Record<string, unknown>, TMenuItem = Record<string, unknown>> = TPanelItem & {
```

Comment:
TMenuItem can probably be dropped and the childen property can simply be an array of TPanelItem since they are the same thing now

Reply:
@drebrez Changes done here https://github.com/neolution-ch/react-pattern-ui/pull/45

Reply:
Changes implemented